specs/DesignDiscussions/InputLabels.md

```diff
@@ -0,0 +1,279 @@
+# Input/Label Associations
+
+This document covers design considerations related to issue [#203](https://github.com/microsoft/AdaptiveCards/issues/203).
+
+Consider the following card:
+```json
+{
+	"type": "AdaptiveCard",
+	"version": "1.0",
+	"body": [
+		{
+			"type": "TextBlock",
+			"text": "Please enter your name:"
+		},
+		{
+			"type": "Input.Text",
+			"id": "nameInput",
+ 			"placeholder": "Last, First"
+		}
+	],
+	"actions": [
+		{
+			"type": "Action.Submit",
+			"title": "OK"
+		}
+	]
+}
+```
+
+![img](assets/InputLabels/InputCard.png)
```

Comment:
This image doesnt render? (Results in a 404 when clicked on also)

Reply:
If you're just looking at the rich diff in the Files Changed tab, the images don't resolve. Try clicking the "..." all the way on the right and then choosing "View file". Let me know if that doesn't work for you.

---
In reply to: [352410831](https://github.com/microsoft/AdaptiveCards/pull/3618#discussion_r352410831) [](ancestors = 352410831)